pkg/action/cobra.go

```diff
@@ -24,7 +24,7 @@ func CobraImpl(a *Action, streams cli.Streams) (*cobra.Command, error) {
 	runOpts := make(TypeOpts)
 	cmd := &cobra.Command{
 		Use:  use,
-		Args: cobra.ExactArgs(len(argsDef)),
+		Args: cobra.MinimumNArgs(1),
```

Comment:
And what if an action doesn't have arguments?

Reply:
removed for now